### ISSUE TITLE
After destroying an API Gateway, upon re-creating it, responses error

### ISSUE THREAD

**Issue by ryanblock** [arc-repos/architect#263]
**Describe the bug**
After destroying an API Gateway, upon re-creating it, responses error out.

**To Reproduce**
1. Basic setup steps:
```
mkdir testapp
cd testapp
npm init --yes
npm install @architect/architect
touch .arc
```
2. Add to .arc:
```
@app
testapp

@http
get /
```
3. Run:
```
npx create && npx deploy
```
4. Hit endpoint – success 👍 
5. Run:
```
npx inventory --nuke
```
6. Confirmed all resources deleted. Then:
```
npx create && npx deploy
```
7. Hit (new) endpoint – `403` 👎 

**Expected behavior**
User should be able to create new resources in the same namespace over and over after destroying it.

**Comment by brianleroux** [comment]
this is a rate limiting error that will be fixed in the ws branch

**Comment by ryanblock** [comment]
I think there's something else going on here, still digging to find out what. Stand by...

**Comment by ryanblock** [comment]
This bug is definitely real, it's definitely mine, and I've got a patch going up in a moment!

**Comment by ryanblock** [comment]
Patched in `4.5.6`, thanks @alexdilley!
